apis/authentication/v1alpha1/openidconnect_webhook.go

```diff
@@ -128,6 +128,29 @@ func (r *OpenIDConnect) validate() field.ErrorList {
 		allErrs = append(allErrs, field.Invalid(field.NewPath("maxTokenExpirationSeconds"), *r.Spec.MaxTokenExpirationSeconds, "should be positive"))
 	}
 
+	if len(r.Spec.ExtraClaims) > 0 {
```

Comment:
I think that we can further simplify this check. Something like reads better IMO. The current check might yield more results than there are in the initial slice [see example in the go playground](https://go.dev/play/p/CqsV7T3cCps)


```golang
	if len(r.Spec.ExtraClaims) > 0 {
		claims := map[string]struct{}{}
		for _, claim := range r.Spec.ExtraClaims {
			lowered := strings.ToLower(claim)
			if _, ok := claims[lowered]; ok {
				allErrs = append(allErrs, field.Invalid(field.NewPath("extraClaims"), r.Spec.ExtraClaims, "duplicated claims found"))
				break
			}
			claims[lowered] = struct{}{}
		}
	}
```

Reply:
Simple is indeed better ! Thanks